documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -398,7 +398,22 @@ This setting is ignored and network install is disabled if `DISABLE_HDMI=1` is s
 
 In order to detect the keyboard, network install must initialise the USB controller and enumerate devices. This increases boot time by approximately 1 second so it may be advantageous to disable network install in some embedded applications.
 
-Default: `1` on Raspberry Pi 4 and Raspberry Pi 400, and `0` on Compute Module 4.
+Default: `1` on Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.
+
+[[NET_INSTALL_AT_POWER_ON]]
+==== `NET_INSTALL_AT_POWER_ON`
+
+When set to `1`, displays the network install UI briefly after a cold boot to make this feature more obvious to new users.
```

Comment:
Presumably this setting is ignored if `NET_INSTALL_ENABLED=0` ? (is that worth mentioning here?)

Reply:
Ah! That's probably worth mentioning too :slightly_smiling_face:

Reply:
That's interesting, I thought it was the other way around although I think I prefer NET_INSTALL_AT_POWER_ON=1 overriding NET_INSTALL_ENABLED

Reply:
```suggestion
When set to `1`, displays the network install UI briefly after a cold boot to make this feature more obvious to new users. Overrides `NET_INSTALL_ENABLED` if the settings conflict.
```